alvr/packets/src/lib.rs

```diff
@@ -59,14 +60,17 @@ impl VideoStreamingCapabilities {
     }
 }
 
+#[derive(Serialize, Deserialize)]
+pub struct ConnectionAcceptedInfo {
+    pub client_protocol_id: u64,
+    pub platform: Platform,
+    pub server_ip: IpAddr,
+    pub streaming_capabilities: Option<VideoStreamingCapabilities>,
+}
+
 #[derive(Serialize, Deserialize)]
 pub enum ClientConnectionResult {
-    ConnectionAccepted {
-        client_protocol_id: u64,
-        display_name: String,
-        server_ip: IpAddr,
-        streaming_capabilities: Option<VideoStreamingCapabilities>,
-    },
+    ConnectionAccepted(Box<ConnectionAcceptedInfo>),
```

Comment:
Why does this need to be boxed?

Reply:
Because clippy complains of big size difference between variants

Reply:
I mean that's not an issue since this is primarily encoded either way, could also disable the clippy warning locally, but not sure if that's a great idea and ig this is fine...